include/nccl_ofi_param.h

```diff
@@ -164,22 +164,12 @@ OFI_NCCL_PARAM_INT(domain_per_thread, "DOMAIN_PER_THREAD", -1);
 
 /*
  * When enabled and RDMA communication protocol is used, write NCCL
- * topology file and set environment variable `NCCL_TOPO_FILE`. By
- * default, OFI plugin writes the NCCL topology file to a unique
- * temporary file using file path template
- * `/tmp/aws-ofi-nccl-topo-XXXXXX` and the file is deleted at normal
- * process termination. See environment variable
- * `OFI_NCCL_TOPO_FILE_TEMPLATE` to control the file destination.
+ * topology file and set environment variable `NCCL_TOPO_FILE`. OFI plugin
+ * writes the NCCL topology file to an anonymous file
+ * and the file is deleted at normal process termination.
```

Comment:
It's not an anonymous file, it's a memfd file.  And since it's just in memory, it doesn't actually get deleted.

Reply:
Reworded